WeatherPredictorApi/SQLApp/crud.py

```diff
@@ -0,0 +1,74 @@
+from xmlrpc.client import Boolean
+from sqlalchemy.orm import Session
+
+from . import database
+
+from .schemas import *
+from . import models
+
+import bcrypt
+
+def recreate_database():
+    database.Base.metadata.drop_all(database.engine)
+    database.Base.metadata.create_all(database.engine)
+
+def getUserByEmail(db: Session, email: str):
+    return db.query(models.user).filter(models.user.email == email).first()
+
+
+def getUserById(db: Session, id : int):
+    return db.query(models.user).filter(models.user.id == id).first()
+
+
+def getAllUsers(db: Session, skip: int = 0, limit: int = 100):
+    return db.query(models.user).offset(skip).limit(limit).all()
+
+def hashpassword(user, salt):
```

Comment:
Why the class that manipulates the database access should know how to hash a password? is it the db's responsibility?